packages/app/src/client/components/withSession/pages/Login.tsx

```diff
@@ -26,15 +25,7 @@ import { Typography } from "client/components/Typography";
 import TracyImg from "client/views/common/Tracy";
 
 export const LoginPage = ({ onLogin }: { onLogin: () => void }) => {
-  useCommandService(
-    {
-      id: "login-on-enter",
-      description: "Login",
-      keybindings: ["enter"],
-      handler: onLogin
-    },
-    []
-  );
+  useHotkeys("enter", onLogin);
```

Comment:
what was the motivation for this change? Just curious since it's not obvious.

Was it something to do with switching the hierarchy of the following?
```
<WithSession>
          <Services>
```